content/docs/sidebar.json

```diff
@@ -299,6 +299,10 @@
           {
             "label": "plots diff",
             "slug": "diff"
+          },
+          {
+            "label": "plots modify",
```

Comment:
do we need to update highlighter?

Reply:
Already in config\prismjs\dvc-commands.js